pkg/oc/bootstrap/docker/up.go

```diff
@@ -324,26 +317,54 @@ func (config *ClientStartConfig) Bind(flags *pflag.FlagSet) {
 	config.CommonStartConfig.Bind(flags)
 }
 
-func (c *CommonStartConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command) error {
+func (c *CommonStartConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command, out io.Writer) error {
 	c.originalFactory = f
 	c.command = cmd
 
+	// do some defaulting
 	if len(c.ImageVersion) == 0 {
 		c.ImageVersion = defaultImageVersion()
 	}
 
-	c.addTask(simpleTask("Checking OpenShift client", c.CheckOpenShiftClient))
+	// do some struct initialization next
+
+	// used for some pretty printing
+	taskPrinter := NewTaskPrinter(getDetailedOut(out))
 
-	c.addTask(conditionalTask("Create Docker machine", c.CreateDockerMachine, func() bool { return c.ShouldCreateDockerMachine }))
+	// create a docker machine if we need one
```

Comment:
@csrwng do we need this codepath? docker for mac kind of replaced the dockermachine

Reply:
We still have the flag. Like anything else, I think we need to deprecate it first and then get rid of it. I agree, it's not the usual path anymore, but docker machine does still exist. But even if you were using docker machine, we should not be trying to create the vm for you.

Reply:
> We still have the flag. Like anything else, I think we need to deprecate it first and then get rid of it. I agree, it's not the usual path anymore, but docker machine does still exist. But even if you were using docker machine, we should not be trying to create the vm for you.

With all our other breakages, does one more on a dev tool like this really matter?

Reply:
Yes :)

Reply:
@csrwng subscribes to https://en.wikipedia.org/wiki/Broken_windows_theory